frontend/src/scenes/web-analytics/webAnalyticsLogic.tsx

```diff
@@ -2290,7 +2290,8 @@ export const webAnalyticsLogic = kea<webAnalyticsLogicType>([
 
     actionToUrl(({ values }) => {
         const stateToUrl = (): string => {
-            const urlParams = new URLSearchParams()
+            const searchParams = { ...router.values.searchParams }
+            const urlParams = new URLSearchParams(searchParams)
```

Comment:
I think this should be okay. I've seen people completely replace it or keep the previous state around on the codebase.

Testing it worked. The issue now was that <CompareFilter> changes would strip the `stripQueryParams` (😅) since it is controlled by `webAnalyticsLogic.`

Reply:
Cool - I trust your testing, we can always tweak it if we get feedback